### PR TITLE
Remove mix color swatches, further fix blend with duplicate sub-materials

### PR DESCRIPTION
**Pull request by usakhelo**
Color swatches are removed. Number amount is texture multiplier when texture presented and material mixing amount without texture.
Also fix blend material in cases when sub-materials have duplicate names. Previous fix failed when sub-material is another blend material with the same name.

### CODE REVIEW

File: src/appleseed-max-impl/appleseedblendmtl/appleseedblendmtl.rc [hunk 1]
```diff
@@ -56,71 +56,62 @@ FONT 8, "MS Sans Serif", 0, 0, 0x1
...
     CONTROL         "Layer Material 1",IDC_MTLBTN_LAYER_1,"CustButton",WS_TABSTOP,18,29,79,10
     CONTROL         "Texture Amount Edit 1",IDC_EDIT_AMOUNT_1,"CustEdit",WS_TABSTOP,179,29,25,10
     CONTROL         "Texture Amount Spinner 1",IDC_SPIN_AMOUNT_1,
                     "SpinnerControl",0x0,205,29,7,10
-    CONTROL         "Mix Color 1",IDC_MIX_AMOUNT_1,"ColorSwatch",WS_TABSTOP,100,29,10,10
     LTEXT           "1:",IDC_LABEL_COAT_1,9,30,8,8
     LTEXT           "Layers:",IDC_LABEL_BASE_MTL2,18,20,24,8
     LTEXT           "Blend Amount:",IDC_LABEL_BASE_MTL3,100,20,52,8
-    CONTROL         "Mask Texture 1",IDC_TEXBTN_MASK_2,"CustButton",WS_TABSTOP,112,42,65,10
+    CONTROL         "Mask Texture 1",IDC_TEXBTN_MASK_2,"CustButton",WS_TABSTOP,100,42,76,10
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedblendmtl/appleseedblendmtl.rc]: Fix control names.

File: src/appleseed-max-impl/oslutils.cpp [hunk 2]
```diff
@@ -353,7 +353,7 @@ void connect_sub_mtl(
         return;
 
     std::string layer_name =
-        make_unique_name(assembly.materials(), wide_to_utf8(mat->GetName()) + "_sub_mat");
+        make_unique_name(assembly.materials(), asf::format("{0}_{1}_{2}", shader_name, mat->GetName(), "sub_mat"));
```
Comment (dictoon) [inline src/appleseed-max-impl/oslutils.cpp]: "sub_mat" can be part of the format string.

**Comment by dictoon** [comment]
Thanks man!

File: src/appleseed-max-impl/oslutils.cpp (hunk 2 above)
Comment (usakhelo) [reply src/appleseed-max-impl/oslutils.cpp]: such a stupid mistake really :)

**Comment by usakhelo** [comment]
Found one more bug in blend material - max crushes when empty blend material is used as a base sub-material. Now it's fixed. And resource file is cleaned up.
Thanks for the review.

File: src/appleseed-max-impl/oslutils.cpp (hunk 2 above)
Comment (dictoon) [reply src/appleseed-max-impl/oslutils.cpp]: It's not even really a mistake :)

**Comment by dictoon** [comment]
Thanks!!

**Pull request merged**